packages/next-starter/postcss.config.js

```diff
@@ -2,6 +2,5 @@ module.exports = {
   plugins: {
     tailwindcss: {},
     autoprefixer: {},
-    'postcss-easy-import': {},
```

Comment:
if this one removed, will next js know how to import the CSS file(@import)?

Reply:
yes, it works well.